.github/workflows/test-platforms.yml

```diff
@@ -0,0 +1,333 @@
+# Run kickstart tests in a PR triggered by a "/test-platforms" command from an organization member
+#
+# /test-platforms - tests all tests affected by the PR or smoke tests if there is no such.
```

Comment:
I would propose to change the comment to `execute all tests...`

Reply:
updated